Makefile

```diff
@@ -1,22 +1,44 @@
-.PHONY: build test test-integration generate install-paramgen
+.PHONY: build test test-integration generate install-paramgen install-mocken
```

Comment:
I noticed we have `PHONY` here, and before each target, shouldn't we only have one approach? or does this have another purpose?

Reply:
Good point, I'll remove those. I think I just copy-pasted this.:)